packages/design-system/src/new-ui/DataTable/DataTable.tsx

```diff
@@ -98,25 +101,43 @@ export function DataTable<TData, TValue>({
           ))}
         </Table.Header>
         <Table.Body>
-          {table.getRowModel().rows?.length ? (
-            table.getRowModel().rows.map((row) => (
-              <Table.Row
-                key={row.id}
-                data-state={row.getIsSelected() && "selected"}
-              >
-                {row.getVisibleCells().map((cell) => (
-                  <Table.Cell key={cell.id}>
-                    {flexRender(cell.column.columnDef.cell, cell.getContext())}
-                  </Table.Cell>
-                ))}
-              </Table.Row>
-            ))
-          ) : (
+          {isLoading ? (
             <Table.Row>
```

Comment:
@iamnamananand996 When in loading we better showcase more rows, let expose a props called `loadingRows` and make its default to 6 to console the amount of the loading roes

Reply:
Hi @EiffelFly , update the suggested change now